.github/scripts/resolve_release_tag.py

```
@@ -0,0 +1,238 @@
+        if not isinstance(release.get("published_at"), str):
+            continue
+        match = STABLE_TAG_PATTERN.fullmatch(str(release.get("tag_name", "")))
+        if match is not None:
+            major, minor, patch = (int(part) for part in match.groups())
+            versions.append((major, minor, patch))
+    return versions
+
+
+def resolve_release_tag(
+    explicit_tag: str,
+    is_prerelease: bool,
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)